### DISCUSSION TITLE
split pocket book to multiple book

### DISCUSSION THREAD

**Discussion by Monirzadeh** [mmagnus/Pocket-Plus-Calibre-Plugin#42] [Ideas]
it take so much time to open a large book on some device (if you have so many link in your pocket)
if it possible please add an option to split final book to multiple book automatically
for example user set every 50 link as new book or set limit that with size of the book

**Comment by mmagnus** [comment]
Dear @Monirzadeh , for now, there is an **indirect** way to limit the size of the big is to use:

     MAX_ARTICLES_PER_FEED = 10

and I think I will not have time to work on something more than this.

Remember, that you can also have a copy of this plugin to set various workflows with different configurations.

And you can use tags to fetch different ebooks, so then you will have a tag-based single book.

<img width="900" alt="more" src="https://user-images.githubusercontent.com/118740/81910658-85dde400-95cc-11ea-857f-47ea67286a1a.png">

**Comment by mmagnus** [comment]
@Monirzadeh  this is exciting... I actually I develop something like this right now.. I forgot about this issue.. and then I looked again, and it's pretty much what I just developed. I will push it in the evening.. stay tuned!

**Comment by mmagnus** [comment]
https://github.com/mmagnus/Pocket-Plus-Calibre-Plugin/blob/master/pocketX/PocketX.recipe

<img width="531" alt="Screen Shot 2020-05-20 at 10 49 11 AM" src="https://user-images.githubusercontent.com/118740/82426029-9d631400-9a87-11ea-91f1-4998296131d3.png">

so it does not produce this sub-books, but you can make on shell script to combine steps into one script
